src/client/scripts/InitHome.ts

```diff
@@ -10,8 +10,141 @@
 module StopByStop {
 
     export class InitHome {
+ 
+        //The functionality below pulls the image url based on lat long
+        //And populates the images inside a div
+        //If an image is not available, it is shown as a blank div
+        public static addImagesDynamically(prevPlace,currentLocationString) {
+            //$('#to').on('click', function (event) {
+            if (AppState.current.app != SBSApp.Web) {    
+                var place = $('#from').data('place');
+                
+                if(place===null){
+                    return true;
+                }
+                
+                if(prevPlace!=null && prevPlace.n===place.n){
+                    
+                    // Dont proceed further as the same place is being selected
+                    return true;
+                }
+
```

Comment:
Can you combine these two checks into one check: if (!place || prevPlace.n === place.n)